inlong-dashboard/src/configs/menus/index.ts

```diff
@@ -38,10 +38,10 @@ const menus: MenuItemType[] = [
     path: '/approvals',
     name: i18n.t('configs.menus.ApprovalManagement'),
   },
-  {
-    path: '/datasources',
-    name: i18n.t('configs.menus.Datasources'),
-  },
+  // {
+  //   path: '/datasources',
+  //   name: i18n.t('configs.menus.Datasources'),
+  // },
```

Comment:
remove them may be better?

Reply:
The menu is temporarily hidden, and the code has other functions.